### DISCUSSION TITLE
Are Transforms Supported?

### DISCUSSION THREAD

**Discussion by westy92** [unicode-org/icu4x#4346] [Q&A]
ICU supports [Transforms](https://unicode-org.github.io/icu/userguide/transforms/general/#overview) like:

```
NFD; [:Nonspacing Mark:] Remove; NFC
```

These are supported using the `rust_icu` package.

```rust
let transliterator = rust_icu_utrans::UTransliterator::new(
    "NFD; [:Nonspacing Mark:] Remove; NFC",
    None,
    rust_icu_sys::UTransDirection::UTRANS_FORWARD,
)
.unwrap();

let parsed = transliterator.transliterate("test").unwrap();
```

Are/will they be supported in ICU4X?

**Comment by sffc** [comment]
We have an `icu_transliterate` component, currently experimental, contributed by @skius.

https://unicode-org.github.io/icu4x/docs/icu_transliterate/struct.RuleCollection.html

CC @robertbastian
